test/verify/check-system-s4u-ssh

```diff
@@ -108,7 +108,7 @@ session    include      system-auth
 """)
 
         sshd_machine.execute(script="""#!/bin/sh
-        sed -i '/\[domain\/cockpit.lan\]/ a pam_gssapi_services = sudo, sudo-i' /etc/sssd/sssd.conf
+        sed -i '/\\[domain\\/cockpit.lan\\]/ a pam_gssapi_services = sudo, sudo-i' /etc/sssd/sssd.conf
```

Comment:
Let's use a raw string here.  This is also not consistent, 5 lines down in the `foobarfoo\nfoobarfoo\n..` are still single quotes.

Reply:
This is kinda funny: we have echo -ne to get the `\n` expanded, but they actually never even made it that far, since Python did it for us....

Using a raw string now, and it will do what was clearly originally intended here.